sirius-header.js

```diff
@@ -1,7 +1,16 @@
 export class SiriusHeader extends HTMLElement {
   connectedCallback() {
-    const prefix =
-      window.location.hostname === "localhost" ? "http://localhost:8080" : "";
+    let prefixFinance
+    const isLocalHost = window.location.hostname === "localhost"
+    let prefix = isLocalHost ? "http://localhost:8080" : "";
+    const financeFlag = this.getAttribute("finance-flag") ?? 0;
+
+    let paymentsLink = '/supervision/#/finance-hub/reporting'
+
+    if (financeFlag === "1") {
+      prefixFinance = isLocalHost ? "http://localhost:8888/finance-admin/downloads" : prefix;
+      paymentsLink = isLocalHost ? "" : "/supervision/finance-admin/downloads";
+    }
```

Comment:
This would break in local environments where the financeFlag isn't set, as prefixFinance isn't being defaulted to anything. If would be simpler just to set the payment URL, e.g. `let paymentsURL = `${prefix}/supervision/#/finance-hub/reporting`` and then override it if the finance flag is set, e.g. 

`paymentsLink = isLocalHost ? "http://localhost:8888/supervision/finance-admin/downloads" : "/supervision/finance-admin/downloads";`

It will probably not work in local envs anyway as Finance Admin isn't going to be running on :8888 but this would be tidier and easier to parse.